src/main/webapp/app/communication/post/post.component.ts

```diff
@@ -217,11 +219,26 @@ export class PostComponent extends PostingDirective<Post> implements OnInit, OnC
         this.pageType = this.metisService.getPageType();
         this.contextInformation = this.metisService.getContextInformation(this.posting);
         this.isAtLeastTutorInCourse = this.metisService.metisUserIsAtLeastTutorInCourse();
+        this.updateShowSearchResultInAnswersHint();
         this.sortAnswerPosts();
         this.assignPostingToPost();
         this.fetchForwardedMessages();
     }
 
+    updateShowSearchResultInAnswersHint() {
+        const searchQuery = this.searchQuery()?.toLowerCase();
+        if (searchQuery === undefined || searchQuery === '') {
```

Comment:
I guess here you could simply use `!searchQuery`, but I think that's up to personal preference